docs/standard/analyzers/portability-analyzer.md

```diff
@@ -7,35 +7,66 @@ ms.assetid: 0375250f-5704-4993-a6d5-e21c499cea1e
+If you want to use [ApiPort console app](https://aka.ms/apiportdownload).
 
 * Type the following command to analyze the current directory: `\...\ApiPort.exe analyze -f .`
 * To analyze a specific list of .dll files, type the following command: `\...\ApiPort.exe analyze -f first.dll -f second.dll -f third.dll`
+* Run `\...\ApiPort.exe -?` to get more help
+
+### View and interpret portability result
+
+Only APIs that are unsupported by a Target Platform appear in the report. 
+After running the analysis in Visual Studio, you'll see your .NET Portability report file link pops up. If you used [ApiPort console app](https://aka.ms/apiportdownload), your .NET Portability report is saved as a file in the format you specified. The default is in an Excel file (*.xlsx*) in your current directory. 
+
+#### Portability Summary 
+
+![Portability Summary](./media/portability-analyzer/PortabilitySummary.PNG)
+
+The Portability Summary section of the report shows the the portability percentage for each assembly included in the run. In the previous example, 89.74% of the .NET Framework APIs used in the `ConsoleAppFramework` app are available in .NET Core + Platform Extensions v2.2. If you run the .NET Portability Analyzer tool against multiple assemblies, each assembly should have a row in the Portability Summary report.
 
-Your .NET Portability Report is saved as an Excel file (*.xlsx*) in your current directory. The **Details** tab in the Excel Workbook contains more information.
+#### Details
+
+![Portability Details](./media/portability-analyzer/PortabilityDetails.PNG)
+
+The Details section of the report lists the APIs missing from one of the Target Platforms. 
+
+ - Target type: the type has missing API from a Target Platform 
+ - Target member: the method is missing from a Target Platform 
+ - Header for assembly name entries: the .NET Framework assembly that the missing
+   API lives in. 
+ - Each of the selected Target Platforms is one column, such as ".NET Core": "Not supported" value means the API is not
+   supported on this Target Platform. 
+ - Recommended Changes: recommended API or technology to change to. Currently, this field is empty or out of date for a lot of APIs. Due to the large number of APIs, we have big challenge to keep it up. We are looking at alternate solutions to provide helpful information to customers.
+
+#### Missing Assemblies
+
+![Portability Details](./media/portability-analyzer/MissingAssemblies.PNG)
+
+You may find a Missing Assemblies section in your report. It has a list of the assemblies in "Header for assembly name entries" column. It normally are the assemblies that your analyzed assembly reference to and they are not .NET Framework assemblies. If it's an assembly that you own, include it in the portability analyze run. If it's third party library, looks for if they have newer version supporting .NET Core or .NET Standard. If so, consider moving to the newer version.  
```

Comment:
The third sentense reads weird. Should may be: These are assemblies that your analyzed assembly references but could not be found by the analyzer.

Also, a comment to the fact that there is nothing wrong with having missing assemblies; they just weren't analyzed. We've discussed renaming it to "Unresolved Assemblies" instead for that reason.

Reply:
Updated to address the feedback.